src/XRP/model/xrp-escrow-cancel.ts

```diff
@@ -20,21 +21,33 @@ export default class XrpEscrowCancel {
   public static from(
     escrowCancel: EscrowCancel,
     xrplNetwork: XrplNetwork,
-  ): XrpEscrowCancel | undefined {
+  ): XrpEscrowCancel {
     const owner = escrowCancel.getOwner()?.getValue()?.getAddress()
     if (!owner) {
-      return undefined
+      throw new XrpError(
+        XrpErrorType.MalformedProtobuf,
+        'EscrowCancel protobuf is missing valid `Owner` field.',
```

Comment:
```suggestion
        'EscrowCancel protobuf is missing valid `owner` field.',
```

Reply:
I think all fields should be lower-case... I know I've missed a few of these in other PRs.  mostly an aesthetic thing, not a big deal and I actually don't care if you fix it, just noticing here due to the inconsistency with this `Owner` and the reference to the same field just below

Reply:
Ah ok, I was capitalizing because that's how it was in the documentation, so I thought that would be better.